htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -956,8 +956,12 @@ public function fetchAll($sortorder = '', $sortfield = '', $limit = 0, $offset =
 		$sql .= " t.date_creation,";
 		$sql .= " t.date_lim_reglement,";
 		$sql .= " t.tms as date_modification,";
-		$sql .= " t.date_export";
+		$sql .= " t.date_export,";
+		$sql .= " c.nom as customer_name,";
+		$sql .= " s.nom as supplier_name";
 		$sql .= ' FROM '.MAIN_DB_PREFIX.$this->table_element.' as t';
+		$sql .= " LEFT JOIN llx_societe as c ON t.subledger_account = c.code_compta";
```

Comment:
This join is not enough. For example with multicompany, you can have sever record in table c with same code_compta.
It will generate duplicates records and all rest of code bugged.
It is better to keep request as it is and fetch the thirdparty after by a fetch, when we need it.